package.json

```diff
@@ -1,6 +1,6 @@
 {
   "devDependencies": {
-    "daisyui": "^4.6.0",
-    "tailwindcss": "^3.4.1"
-  }
+    "daisyui": "^4.7.3"
+  },
+  "packageManager": "pnpm@8.15.4"
```

Comment:
The latest version of `pnpm` adds this when I install or update deps for some reason.